### PR TITLE
Add rm-cluster command to remove a cluster config

### PR DESCRIPTION
**Pull request by gerep**
The `rm-cluster` command receives the cluster name as the parameter, removes
it from the map variable and rewrites the config file with the remaining
clusters.

Closing #504

### CODE REVIEW

File: pkg/client/cmd/config.go [hunk 1]
```diff
@@ -159,6 +172,38 @@ func setCluster(cmd *cobra.Command, args []string) {
...
+		return
+	}
+
+	c, err := client.ReadConfigFile(cfgFile)
+	if err != nil {
+		c = &client.Config{
+			Clusters:       make(map[string]client.ClusterConfig),
+			CurrentCluster: name,
+		}
+	}
```
Comment (aguerra) [inline pkg/client/cmd/config.go]: You guess you don't need to handle this case for rm-cluster, just PrintErrorAndExit.

Comment (gerep) [reply pkg/client/cmd/config.go]: @aguerra good point. Something like this instead?

`client.PrintErrorAndExit("Error trying to read config file: %v", err)`

**Comment by gerep** [comment]
@aguerra is there anything else I can do so we can merge this PR?

File: pkg/client/cmd/config.go [hunk 2]
```diff
@@ -159,6 +172,35 @@ func setCluster(cmd *cobra.Command, args []string) {
 	}
 }
 
+// rmCluster removes a server from the config file
+func rmCluster(cmd *cobra.Command, args []string) {
+	if len(args) == 0 {
```
Comment (drgarcia1986) [inline pkg/client/cmd/config.go]: What about?:
```go
if len(args) == 0 || args[0] == "" {
    cmd.Usage()
    return
}
```

File: pkg/client/cmd/config.go [hunk 3]
```diff
@@ -159,6 +172,35 @@ func setCluster(cmd *cobra.Command, args []string) {
...
+	if err != nil {
+		client.PrintErrorAndExit("Error trying to read config file: %v", err)
+	}
+
+	if _, found := c.Clusters[name]; found {
+		delete(c.Clusters, name)
+		if err = client.SaveConfigFile(cfgFile, c); err != nil {
+			client.PrintErrorAndExit("Error trying to save config file: %v", err)
+		}
+		fmt.Printf("Config %s removed", name)
```
Comment (drgarcia1986) [inline pkg/client/cmd/config.go]: Could you add a `\n` in end of the text?

**Comment by aguerra** [comment]
Hi @gerep, sorry about the delay (I've disabled github notifications). Now we are back in business so it shouldn't take long to merge it.

File: pkg/client/cmd/config.go [hunk 4]
```diff
@@ -159,6 +172,35 @@ func setCluster(cmd *cobra.Command, args []string) {
...
 
+// rmCluster removes a server from the config file
+func rmCluster(cmd *cobra.Command, args []string) {
+	if len(args) == 0 || args[0] == "" {
+		cmd.Usage()
+		return
+	}
+
+	name := args[0]
+	if name == "" {
```
Comment (drgarcia1986) [inline pkg/client/cmd/config.go]: Hm, I think you don't need to check if `args[0]` is empty again, you already did that on line 177.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [NEXT_RELEASE]
+### Added
+- [client] Add support to remoVe remove cluster from configuration file
```
Comment (aguerra) [inline CHANGELOG.md]: remoVe remove -> remove

Comment (gerep) [reply CHANGELOG.md]: Yes, noticed that a bit late, sorry.

**Pull request merged**